libibmad/register.c

```diff
@@ -117,15 +117,15 @@ int mad_register_client_via(int mgmt, uint8_t rmpp_version,
 }
 
 int mad_register_server(int mgmt, uint8_t rmpp_version,
-			long method_mask[], uint32_t class_oui)
+			long method_mask[16 / sizeof(long)], uint32_t class_oui)
```

Comment:
Why not just use "long method_mask *" ?

Reply:
It still produces same warning of differences between .c and .h. We can't change .h because it is ABI.